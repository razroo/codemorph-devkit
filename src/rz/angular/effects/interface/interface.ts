import { exportTsFiles, isTsFile } from "../../../utils/add-export";

export function exportInterfaceFile(filePathWithName: string): void {
  if (isTsFile(filePathWithName)) {
    exportTsFiles(filePathWithName);
  }
}