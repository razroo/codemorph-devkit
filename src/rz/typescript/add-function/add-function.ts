import { EditCodeBlockInput } from "../interfaces/edit-typescript.interface";

export function addFunction(editCodeBlockInput: EditCodeBlockInput): void {
    const sourceFile = editCodeBlockInput.sourceFile;

    sourceFile.addFunction({
      name: editCodeBlockInput.name,
      returnType: editCodeBlockInput.type,
      statements: editCodeBlockInput.codeBlock,
      isExported: editCodeBlockInput.isExported,
      parameters: editCodeBlockInput.parameters
    });
}