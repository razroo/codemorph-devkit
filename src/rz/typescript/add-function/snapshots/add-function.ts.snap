export function pushScaffoldCommands(context, isProduction: boolean, packageJsonParams) {
}